whois-client/src/main/java/net/ripe/db/whois/common/sso/AuthServiceClient.java

```diff
@@ -162,6 +165,32 @@ public ValidateTokenResponse getUserDetails(final String uuid) {
         }
     }
 
+    @Cacheable(cacheNames="ssoHistoricalUserDetails", key="#uuid")
+    public HistoricalUserResponse getHistoricalUserDetails(final String uuid) {
+        if (StringUtils.isEmpty(uuid)) {
+            LOGGER.debug("No uuid was supplied");
+            throw new AuthServiceClientException(BAD_REQUEST.getStatusCode(),"Invalid uuid.");
+        }
+
+        try {
+            return client.target(restUrl)
+                    .path(HISTORICAL_USER_SEARCH_PATH)
+                    .path(uuid)
+                    .request(MediaType.APPLICATION_JSON_TYPE)
+                    .header("X-API_KEY", apiKey)
```

Comment:
Is the query parameter `format=json` not necessary?

Reply:
no, it is not required (tested with insomnia)